src/main/java/lotto/Rank.java

```diff
@@ -0,0 +1,18 @@
+package lotto;
+
+public enum Rank {
+    MISS_MATCH(0,0),
+    MATCH_3(3,5000 ),
+    MATCH_4(4,50000),
+    MATCH_5(5,1500000),
+    MATCH_5_BONUS(5,30000000),
```

Comment:
```suggestion
    MATCH_5_BONUS(5, 30_000_000),
```
java에서 큰수의 가독성을 높이기 위해 `30_000_000` 이렇게 언더스코어(`_`)로 숫자를 잘라서 표현이 가능합니다.
참고해주세요!🤗

그리고 코드컨벤션에서 띄워쓰기 부탁드립니다.

<참고>
> IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L), import 최적화 기능(⌃⌥O, Ctrl+Alt+O)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요. 패키지를 선택한 다음 단축키를 눌러 보세요🤗
그리고 IntelliJ IDEA에서 [google code style 적용](https://toma0912.tistory.com/93)이나 [google code style plugin](https://plugins.jetbrains.com/plugin/8527-google-java-format)을 설치하셔서 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 하시면 자동으로 반영되니, 리뷰요청전 커밋에 반영해서 적용해주세요! 🙇‍♂️